### PR TITLE
Core Team Issue templates

### PR DESCRIPTION
**Pull request by Himangini**
### Description
Added two new issue templates for Spike and User story [to be used by Core team ] 
Also, slightly modified current templates to have a placeholder title and nicer looking headings

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/core-team-story.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### User story
+<!--
+As a User I want ....
+-->
+
+### Acceptance Criteria
+[] fizz
+[] buzz
+[] fizzbuzz
```
Comment (aclevername) [inline .github/ISSUE_TEMPLATE/core-team-story.md:28]:
```suggestion
- Given X 
- When Y 
- Then Z 
```
example:
Given a an EKS cluster
When I run `eksctl utils update-coredns`
Then the version of coredns running in the cluster should be updated to the latest

Comment (Himangini) [reply .github/ISSUE_TEMPLATE/core-team-story.md]: updated 👍🏻

**Review by Skarlso — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/core-team-spike.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Spike [Core team only - DONOT USE FOR COMMUNITY ISSUES]
```
Comment (nikimanoledaki) [inline .github/ISSUE_TEMPLATE/core-team-spike.md:2]:
```suggestion
name: Spike [Core team only - DO NOT USE FOR COMMUNITY ISSUES]
```

File: .github/ISSUE_TEMPLATE/core-team-story.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+name: Story [Core team only - DONOT USE FOR COMMUNITY ISSUES]
```
Comment (nikimanoledaki) [inline .github/ISSUE_TEMPLATE/core-team-story.md:2]:
```suggestion
name: Story [Core team only - DO NOT USE FOR COMMUNITY ISSUES]
```

File: .github/ISSUE_TEMPLATE/core-team-story.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+### Acceptance Criteria
+<!--
+Acceptance Criteria for the user story
+Eg., 
+Given ...
+When ...
+Then ...
+-->
+
+### Techinical Notes
```
Comment (nikimanoledaki) [inline .github/ISSUE_TEMPLATE/core-team-story.md:34]:
```suggestion
### Technical Notes
```

**Review by nikimanoledaki — APPROVED** [review]
A couple of edits then LGTM! :)

File: .github/ISSUE_TEMPLATE/core-team-story.md (hunk 4 above)
Comment (Himangini) [reply .github/ISSUE_TEMPLATE/core-team-story.md:34]: nice spot ⭐

File: .github/ISSUE_TEMPLATE/core-team-spike.md (hunk 2 above)
Comment (Himangini) [reply .github/ISSUE_TEMPLATE/core-team-spike.md]: done 👍🏻

File: .github/ISSUE_TEMPLATE/core-team-story.md (hunk 3 above)
Comment (Himangini) [reply .github/ISSUE_TEMPLATE/core-team-story.md]: done 👍🏻

File: .github/ISSUE_TEMPLATE/core-team-story.md (hunk 4 above)
Comment (Himangini) [reply .github/ISSUE_TEMPLATE/core-team-story.md]: done 👍🏻

**Pull request merged**